pkg/ccl/changefeedccl/changefeed_dist.go

```diff
@@ -523,6 +524,18 @@ func makePlan(
 		p.PlanToStreamColMap = []int{1, 2, 3}
 		sql.FinalizePlan(ctx, planCtx, p)
 
+		// Log the plan diagram URL so that we don't have to rely on it being in system.job_info.
+		flowSpecs := p.GenerateFlowSpecs()
+		if _, diagURL, err := execinfrapb.GeneratePlanDiagramURL(
+			fmt.Sprintf("changefeed: %d", jobID),
+			flowSpecs,
+			execinfrapb.DiagramFlags{},
+		); err != nil {
+			log.Warningf(ctx, "failed to generate changefeed plan diagram: %s", err)
+		} else {
```

Comment:
You could, a conservative backstop, additionally check that the string you're about to log is under 1mb or something just in case there's a a change in the flow protos at some point that makes them unexpected large again /2¢